package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -163,20 +184,26 @@ export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProv
       ? steps[current]
       : undefined;
 
-    return step ?? { position: Position.BOTTOM, render: () => <></> };
+    return step ?? { floatingProps: { placement: "bottom" }, render: () => <></> };
```

Comment:
I think we can leave the default value here:
```suggestion
    return step ?? { floatingProps, render: () => <></> };
```